lib/chewy/index/actions.rb

```diff
@@ -221,6 +221,10 @@ def journal
           @journal ||= Chewy::Journal.new(self)
         end
 
+        def clear_cache(args = {index: 'index_name'})
+          client.indices.clear_cache(args)
+        end
```

Comment:
It would be nice to have here as default index the name of given index, so imagine that you should be able to run only `CitiesIndex.clear_cache` and that should be enough